concent_api/core/transfer_operations.py

```diff
@@ -108,13 +107,25 @@ def create_file_transfer_token(
     """
     Function to create FileTransferToken from ReportComputedTask message
     """
-    current_time    = get_current_utc_timestamp()
     task_id         = report_computed_task.task_to_compute.compute_task_def['task_id']
     subtask_id      = report_computed_task.task_to_compute.compute_task_def['subtask_id']
     file_path       = 'blender/result/{}/{}.{}.zip'.format(task_id, task_id, subtask_id)
 
+    assert operation in ['upload', 'download']
+    if operation == 'upload':
+        token_expiration_deadline = (
+            report_computed_task.task_to_compute.compute_task_def['deadline'] +
+            3 * settings.CONCENT_MESSAGING_TIME +
+            2 * settings.MAXIMUM_DOWNLOAD_TIME
+        )
+    if operation == 'download':
```

Comment:
Please do `elif` here. It won't change anything in logic but will look more readable, meaning that the code always goes either this way or another.

Reply:
Done